[2025-06-26T20:50:11.111+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-26T20:50:11.133+0700] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: start_kafka_hadoop_and_run_notebooks.start_zookeeper manual__2025-06-26T13:50:08.571246+00:00 [queued]>
[2025-06-26T20:50:11.140+0700] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: start_kafka_hadoop_and_run_notebooks.start_zookeeper manual__2025-06-26T13:50:08.571246+00:00 [queued]>
[2025-06-26T20:50:11.140+0700] {taskinstance.py:2303} INFO - Starting attempt 1 of 4
[2025-06-26T20:50:11.158+0700] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): start_zookeeper> on 2025-06-26 13:50:08.571246+00:00
[2025-06-26T20:50:11.166+0700] {standard_task_runner.py:63} INFO - Started process 161959 to run task
[2025-06-26T20:50:11.171+0700] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'start_kafka_hadoop_and_run_notebooks', 'start_zookeeper', 'manual__2025-06-26T13:50:08.571246+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/start_DAG.py', '--cfg-path', '/tmp/tmpetwi1d1q']
[2025-06-26T20:50:11.172+0700] {standard_task_runner.py:91} INFO - Job 74: Subtask start_zookeeper
[2025-06-26T20:50:11.248+0700] {task_command.py:426} INFO - Running <TaskInstance: start_kafka_hadoop_and_run_notebooks.start_zookeeper manual__2025-06-26T13:50:08.571246+00:00 [running]> on host Panda
[2025-06-26T20:50:11.352+0700] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='group_03' AIRFLOW_CTX_DAG_ID='start_kafka_hadoop_and_run_notebooks' AIRFLOW_CTX_TASK_ID='start_zookeeper' AIRFLOW_CTX_EXECUTION_DATE='2025-06-26T13:50:08.571246+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-26T13:50:08.571246+00:00'
[2025-06-26T20:50:11.354+0700] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-26T20:50:11.355+0700] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-06-26T20:50:11.355+0700] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n        cd /home/panda/kafka_2.13-3.9.1 &&\n        nohup bin/zookeeper-server-start.sh config/zookeeper.properties > /dev/null 2>&1 &\n\n        echo "üïí Waiting for ZooKeeper..."\n\n        for i in {1..10}; do\n            if nc -z localhost 2181; then\n                echo "‚úÖ ZooKeeper is up."\n                exit 0\n            fi\n            sleep 2\n        done\n\n        echo "‚ùå ZooKeeper failed to start in 20s"\n        exit 1\n        ']
[2025-06-26T20:50:11.375+0700] {subprocess.py:86} INFO - Output:
[2025-06-26T20:50:11.377+0700] {subprocess.py:93} INFO - üïí Waiting for ZooKeeper...
[2025-06-26T20:50:13.385+0700] {subprocess.py:93} INFO - Connection to localhost (127.0.0.1) 2181 port [tcp/*] succeeded!
[2025-06-26T20:50:13.385+0700] {subprocess.py:93} INFO - ‚úÖ ZooKeeper is up.
